test/verify/check-storage-btrfs

```diff
@@ -499,6 +499,18 @@ class TestStorageBtrfs(storagelib.StorageCase):
         self.confirm()
         b.wait_visible(self.card_row("Storage", location=f"{subvol_mount_point} (not mounted)"))
 
+        # Make sure we still have the entries in fstab
+        m.execute(f"findmnt --fstab -n {mount_point}")
+        m.execute(f"findmnt --fstab -n {subvol_mount_point}")
+
+        # Delete the whole thing
+        self.click_dropdown(self.card_row("Storage", name=label), "Delete filesystem")
+        self.confirm()
```

Comment:
Should this also do `wipefs` or similar? I.e. could this test `blkid` that the btrfs file system is really gone?

Reply:
Yes, I'll add that and rebase.

Reply:
Great, thank you!